src/shopware_api_client/base.py

```diff
@@ -101,7 +101,15 @@ async def _make_request(self, method: str, relative_url: str, **kwargs: Any) ->
 
         retry_count = 0
         while True:
-            response = await client.request(method, url, headers=headers, **kwargs)
+            try:
+                response = await client.request(method, url, headers=headers, **kwargs)
+            except httpx.RequestError:
+                if retry_count == retries:
+                    raise
```

Comment:
Wollen wir hier die httpx exceptions noch in einen SWAPIError wrappen?

Reply:
Ja, gern.